README.md

```diff
@@ -92,6 +100,22 @@ This project requires contributors to agree to a [Contributor License Agreement
 
 Note that only for contributions to the garm repository on the [GitHub](https://github.com/yahoojapan/garm), the contributors of them shall be deemed to have agreed to the CLA without individual written agreements.
 
+## About releases
+
+- Merging PR to master will increase the version no. and create a new release automatically.
```

Comment:
Do you need to write that if PR title doesn't contain these prefixes, versioning in PR will fail?
This part is affected.
- https://github.com/yahoojapan/authorization-proxy/blob/master/.circleci/config.yml#L170

Reply:
README is for user. I think it is better to add in PR template.
https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

Reply:
>  I think it is better to add in PR template.

ok

> README is for user.

"About releases" describes the commit message.
Is this for user?

Reply:
moved to `CONTRIBUTING.md`